defaults/main.yml

```diff
@@ -2,8 +2,17 @@
 # defaults file for ansible-coreos-kubelet
 coreos_kubelet_systemd: /etc/systemd/system
 coreos_kubelet_install_dir: /opt/bin
-kubelet_version: "v1.4.1_coreos.0"
+kubelet_version: "v1.7.0_coreos.0"
 kubelet_aci: "quay.io/coreos/hyperkube"
```

Comment:
No, this is only an ansible variable. We set Kubelet environment variables in `templates/kube.env.j2`